hadoop-hdds/container-service/src/main/java/org/apache/hadoop/ozone/container/common/interfaces/Container.java

```diff
@@ -57,6 +57,13 @@ void create(VolumeSet volumeSet, VolumeChoosingPolicy volumeChoosingPolicy,
    */
   void delete() throws StorageContainerException;
 
+  /**
+   * Returns true if container is empty
+   * @return true of container is empty
+   * @throws IOException if was unable to check container status.
+   */
+  boolean isContainerEmpty() throws IOException;
```

Comment:
nit: maybe just `isEmpty`, it's already in the container context.

Reply:
Makes sense